### PR TITLE
Clarify 16.1 in Create Term Definition to satisfy #241.

### PR DESCRIPTION
**Pull request by gkellogg**
@kasei, please let us know if this satisfies issue #241.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-api/pull/247.html" title="Last updated on Dec 18, 2019, 6:04 PM UTC (0255501)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-api/247/dd3dc99...0255501.html" title="Last updated on Dec 18, 2019, 6:04 PM UTC (0255501)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6887,6 +6888,15 @@ <h4>JsonLdErrorCode</h4>
     <li>Ignore terms and IRIs that have the form of a keyword (`"@"1*ALPHA`).</li>
   </ul>
 </section>
+<section class="appendix informative" id="changes-from-cr">
+  <h2>Changes since Candidate Release of 12 November 2019</h2>
```
Comment (iherman) [inline index.html]: Should be 12 December :-)

**Comment by gkellogg** [comment]
If we just do editorial updates, then I think we can just track cumulative changes since the first CR. Of course, we can decide to do something differently in the future, but breaking it up into sub-appendices for each interim update might not be helpful.

**Comment by iherman** [comment]
> If we just do editorial updates

Hopefully, that will be the case:-)

We shall see. As I said, it is a minor issue.

**Comment by kasei** [comment]
I would have preferred a branching style as suggested in #241, and used throughout every algorithm in the API document. This is the only place in the entire document that talks about "skipping" "sub-steps". If there is a reason this language was chosen, I can live with it, but it sticks out as being different (and less explicit) than semantically similar steps used elsewhere in the document.

**Comment by gkellogg** [comment]
I was looking to be less disruptive to the algorithmic flow, because this is intended to be a simple edit, rather than a substantial change to the algorithm. I was concerned that grouping the rest of the steps under an encompassing "Otherwise" might look too heavyweight; otherwise, that's what I would normally have done.

But, I'm willing to reconsider pending input from the WG.

**Comment by davidlehn** [comment]
FWIW, I had the same thought as @kasei when I first saw the patch.  General pattern elsewhere is "if(cond) { steps }", but this is more like "if(cond) { return }".  That's fine in code, but a bit odd in prose.

**Pull request merged**